src/section.rs

```diff
@@ -91,7 +91,7 @@ pub trait WritePiggybackSection {
     {
         let section = Section {
             name: Self::name().into(),
-            content: serde_json::to_string(&self).unwrap(),
```

Comment:
I believe this should actually stay an unwrap, since it the function should only error, if serde_json was used incorrectly. Thus, the error needs to stay unrecoverable (but maybe a bit more context could be good).

Reply:
Done